<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSetupDialog</class>
 <widget class="QDialog" name="SerialSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>184</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Serial Setup Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/images/src/serial_tool/gui/images/SerialTool.png</normaloff>:/images/src/serial_tool/gui/images/SerialTool.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="bytesize">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Data size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_dataSize_eight">
         <property name="text">
          <string>8</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">RB_dataSizeGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_dataSize_seven">
         <property name="text">
          <string>7</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">RB_dataSizeGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_dataSize_six">
         <property name="text">
          <string>6</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">RB_dataSizeGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_dataSize_five">
         <property name="text">
          <string>5</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">RB_dataSizeGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="4">
      <layout class="QVBoxLayout" name="parity">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_parity_none">
         <property name="text">
          <string>NONE</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">RB_parityGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_parity_even">
         <property name="text">
          <string>EVEN</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">RB_parityGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_parity_odd">
         <property name="text">
          <string>ODD</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">RB_parityGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="6">
      <layout class="QVBoxLayout" name="stop_bits">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Stop bits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_stopBits_one">
         <property name="text">
          <string>1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">RB_stopBitsGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_stopBits_two">
         <property name="text">
          <string>2</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">RB_stopBitsGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="flow_control">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Software flow control:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_swFlowCtrl">
         <property name="text">
          <string>XON/XOFF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Hardware flow control:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_hwFlowCtrl">
         <property name="text">
          <string>RTS/CTS</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PB_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="RB_stopBitsGroup"/>
  <buttongroup name="RB_dataSizeGroup"/>
  <buttongroup name="RB_parityGroup"/>
 </buttongroups>
</ui>
